emcc.py

```diff
@@ -1342,6 +1342,16 @@ def default_setting(name, new_default):
   if settings.DISABLE_EXCEPTION_THROWING and not settings.DISABLE_EXCEPTION_CATCHING:
     exit_with_error("DISABLE_EXCEPTION_THROWING was set (probably from -fno-exceptions) but is not compatible with enabling exception catching (DISABLE_EXCEPTION_CATCHING=0). If you don't want exceptions, set DISABLE_EXCEPTION_CATCHING to 1; if you do want exceptions, don't link with -fno-exceptions")
 
+  # We cannot enable Emscripten SjLj and Wasm SjLj at the same time
+  if settings.SUPPORT_LONGJMP and settings.SJLJ_HANDLING:
```

Comment:
`SUPPORT_LONGJMP` is enabled by default in `settings.py`.     Perhaps the best thing to do would be to disabled it by default when `SJLJ_HANDLING` is enaled.

I think the way to do this would be to add these two lines just before this check:

```
if settings.SJLJ_HANDLINGL
  default_settings('SUPPORT_LONGJMP', 0)
```

On the other hand if you do that it might confuse code like this in library.js:
https://github.com/emscripten-core/emscripten/blob/f099e309d6ab1b2ec6e1967a72d5e4e90ea19abf/src/library.js#L1498

Would it make sense to instead say that `SUPPORT_LONGJMP` means that longjmp support is enabled in one way or another (i.e. SJLJ_HANDLING implies SUPPORT_LONGJMP)?